app/src/services/account.services.ts

```diff
@@ -2,6 +2,28 @@ import axios from "axios";
 
 import { BACKEND_URL } from "../helpers/environment";
 
+const deleteAccount = async () => {
+    const creatorId = localStorage.getItem("id");
+
+    if(creatorId == null) {
+        throw new Error("No creatorId found in localStorage");
+    }
+
+    try {
+      const res = await axios.delete(
+        `${BACKEND_URL}/profile/delete/${creatorId}`);
```

Comment:
This URL is not following the guidelines, but it should be updated to account for the new user model anyway - I'll let it pass for now and make an issue for this to be fixed since the user change will not be merged until after the review

Reply:
+1, no changes needed, as Jacob intends to break the whole application next sprint